src/components/EditorsColumn.jsx

```diff
@@ -13,58 +17,99 @@ function allErrorsFor(language, errors, runtimeErrors) {
+  return editorRefs.map(ref =>
+    ref ? ReactDOM.findDOMNode(ref).offsetHeight : null);
+}
+
+export default class EditorsColumn extends React.Component {
+  constructor(props) {
+    super(props);
+    bindAll(this, '_storeEditorRef', '_handleEditorDividerDrag');
+  }
+
+  _storeEditorRef(index, editor) {
+    this.editorRefs[index] = editor;
+  }
+
+  _handleEditorDividerDrag(index, _, {deltaY, lastY, y}) {
+    const {onUpdateFlex} = this.props;
+    const heights = extractNodeHeights(this.editorRefs);
+    onUpdateFlex({deltaY, heights, index, lastY, y});
+  }
 
-    editors.push(
-      <EditorContainer
-        key={language}
-        language={language}
-        source={currentProject.sources[language]}
-        onMinimize={
-          partial(onComponentMinimize, `editor.${language}`)
-        }
-      >
-        <Editor
-          errors={allErrorsFor(language, errors, runtimeErrors)}
+  render() {
+    const {
+      currentProject,
+      editorsFlex,
+      errors,
+      onComponentMinimize,
+      onEditorInput,
+      onRequestedLineFocused,
+      runtimeErrors,
+      ui,
+    } = this.props;
+
+    const editors = [];
+    this.editorRefs = [];
+    const languages = ['html', 'css', 'javascript'].filter(language =>
+      !includes(ui.minimizedComponents, `editor.${language}`),
+    );
+    languages.forEach((language, index) => {
+      editors.push(
+        <EditorContainer
           key={language}
           language={language}
-          percentageOfHeight={1 / editors.length}
-          projectKey={currentProject.projectKey}
-          requestedFocusedLine={ui.editors.requestedFocusedLine}
           source={currentProject.sources[language]}
-          onInput={partial(onEditorInput, language)}
-          onRequestedLineFocused={onRequestedLineFocused}
-        />
-      </EditorContainer>,
-    );
-  });
+          style={{flex: editorsFlex[index], minHeight: EDITOR_MIN_HEIGHT}}
+          onMinimize={
+            partial(onComponentMinimize, `editor.${language}`)
+          }
+        >
+          <Editor
+            errors={allErrorsFor(language, errors, runtimeErrors)}
+            key={language}
+            language={language}
+            percentageOfHeight={1 / languages.length}
+            projectKey={currentProject.projectKey}
+            ref={partial(this._storeEditorRef, index)}
+            requestedFocusedLine={ui.editors.requestedFocusedLine}
+            source={currentProject.sources[language]}
+            onInput={partial(onEditorInput, language)}
+            onRequestedLineFocused={onRequestedLineFocused}
+          />
+          {(index < languages.length - 1) &&
+            <DraggableCore
+              axis="y"
+              bounds="parent"
+              key={`divider:${language}`}
+              onDrag={partial(this._handleEditorDividerDrag, index)}
+            >
+              <div
+                className="editors__divider"
+                style={{height: DIVIDER_HEIGHT}}
```

Comment:
@outoftime that *might* work, and if it does then I'll be happy to use it.

I'm slightly worried that this could turn into one of those times where one needs a property of a rendered thing before it's rendered; in this case, per the discussion about needing the divider's height in order to set the default flex properties of the editor containers, it's possible we need the height of the divider before we've rendered the divider's container.

I can think of a few hacky workarounds for that 
- Say "screw it" to equal default editor heights and allow the top two editors to be slightly smaller (again see discussion about default flex)
- The above hack but then adjusting as soon as we get the divider height

I'll note that I believe this issue only applies to the divider height; I can't think of any trouble using `getComputedStyle` for the editor min height.

Thoughts?